### DISCUSSION TITLE
Get value from Relationship field dropdown and use in another field

### DISCUSSION THREAD

**Discussion by wolseyt** [Budibase/budibase#12948] [Help]
Hello
I am trying to extract the value from the relationship field (Equip_Type) and use it to automatically add this value to another text field (equiptype). As you can see from the screen shot, the field (equiptype) seems to be automatically filled with some strange id value when I update the selection of the relationship field. 
In the screen shot example, both "equiptype" and "Equip_Type should show the value "INSTRUMENTS".

![image](https://github.com/Budibase/budibase/assets/113090083/f10192fe-fdb5-45a4-a708-192df7ac82d2)

thank you
Tony

**Comment by ConorWebb96** [comment]
Hi Tony,

I know you've emailed into support, but just copying across the response here incase it helps anyone else who has a similar issue.

That does seem to be getting the correct value. The relationship field's value retrieves the related id of the record. What you are seeing is the primary key id of the row encoded.

I'm assuming you are using an update field action to set this value. What you can do is called action changing. Basically you would grab the field value of the relationship picker then use this with the fetch row action to get the specific row back. Then use the fetch row results to update the field with the `row.Namerow.nameOfField`. This would then set the name or other relevant field to this one. (I've attached some screenshots to help with this explanation below.)

<img width="1512" alt="Screenshot 2024-02-05 at 09 03 47" src="https://github.com/Budibase/budibase/assets/126772285/9481db10-7f41-4e72-b64e-bb0a67da4215">
<img width="1512" alt="Screenshot 2024-02-05 at 09 04 04" src="https://github.com/Budibase/budibase/assets/126772285/dc7d004d-8156-4dc0-813b-375f6ff83ba5">
<img width="1512" alt="Screenshot 2024-02-05 at 09 04 12" src="https://github.com/Budibase/budibase/assets/126772285/dacfe18f-cd58-489d-9183-2e47cc4abc01">
